public/app/features/canvas/elements/button.tsx

```diff
@@ -28,7 +28,7 @@ class ButtonDisplay extends PureComponent<CanvasElementProps<ButtonConfig, Butto
     };
 
     return (
-      <Button type="submit" onClick={onClick}>
+      <Button type="submit" onClick={onClick} style={{ background: defaultColor }}>
```

Comment:
Not a big deal if we end up black listing button as a selectable value but it might be nice to have the button stay centered vertically / horizontally in the element container
![Screen Shot 2022-07-29 at 12 10 25 PM](https://user-images.githubusercontent.com/22381771/181828520-2225c30a-f2bf-4476-99ff-99add9047140.png)

Reply:
I removed the button for now, but updated the size to 32x32 to match the button, so it aligns with the rest of the elements.